Libs/RaptureCore/src/test/java/rapture/kernel/scripting/ScriptInterfaceTest.java

```diff
@@ -76,7 +80,17 @@ public void testInputProgram() throws RecognitionException {
         //assertTrue(testScript("println('hello');println(_params['hello']);return 5;\n").isEmpty());
     	
         ReflexRaptureScript rrs = new ReflexRaptureScript();
-        String test = getResourceAsString(this, "/scriptinterface.script");
+        String test = "";
```

Comment:
If you return empty string instead of null or throwing an exception then the test case will pass, because an empty string is considered a valid Reflex script.